radio/src/targets/taranis/hal.h

```diff
@@ -778,8 +778,10 @@
     #define ADC_CHANNEL_POT3            ADC_Channel_9
     #define ADC_VREF_PREC2              330
   #else
+    #define ADC_GPIO_PIN_POT3           GPIO_Pin_11  // PD.11
     #define ADC_GPIOB_PINS              (ADC_GPIO_PIN_POT2)
-    #define ADC_CHANNEL_POT3            0
+    #define ADC_GPIOD_PINS              (ADC_GPIO_PIN_POT3)
+    #define ADC_CHANNEL_POT3            ADC_Channel_9
```

Comment:
If I reset this to `0` then the 7th analog (on the analog debug screen) follows the reverse of the elevator stick.